tests/virtualization/universal/hotplugging_network_interfaces.pm

```diff
@@ -54,24 +53,29 @@ sub add_virtual_network_interface {
 
 sub run_test {
     my ($self) = @_;
+    my @guests = keys %virt_autotest::common::guests;
     my ($sles_running_version, $sles_running_sp) = get_os_release;
 
     record_info "SSH", "Check if guests are online with SSH";
-    wait_guest_online($_) foreach (keys %virt_autotest::common::guests);
+    wait_guest_online($_) foreach (@guests);
 
     # Add network interfaces
     my %mac = ();
     record_info "Virtual network", "Adding virtual network interface";
-    $mac{$_} = add_virtual_network_interface($self, $_) foreach (keys %virt_autotest::common::guests);
+    set_var("VIRT_AUTOTEST", "1");
+
+    $mac{$_} = add_virtual_network_interface($self, $_) foreach (@guests);
+    set_var("VIRT_AUTOTEST", "");
```

Comment:
Logic changed here ? From set_var("VIRT_AUTOTEST", "1"); to set_var("VIRT_AUTOTEST", ""); ?

Reply:
This is temporary workaround for a bug. will change back.